playground/README.md

```diff
@@ -0,0 +1,153 @@
+# Fancy Regex Playground
+
+A browser-based interactive playground for testing and exploring the [fancy-regex](https://github.com/fancy-regex/fancy-regex) crate. This playground is compiled to WebAssembly (WASM) and provides a user-friendly interface for experimenting with advanced regex features like backreferences and lookaround.
+
+## ✨ Features
+
+- **Interactive Regex Testing**: Enter regex patterns and test text to see real-time results
+- **Advanced Regex Support**: Full support for fancy-regex features including:
+  - Backreferences (`\1`, `\2`, etc.)
+  - Named capture groups (`(?<name>...)` and `\k<name>`)
+  - Lookahead and lookbehind assertions
+  - All other fancy-regex features
+- **Real-time Highlighting**: Visual highlighting of matches and capture groups in the test text
```

Comment:
Having tried it out, it doesn't actually highlight capture groups, only matches?